py/torch_tensorrt/dynamo/conversion/impl/argmax.py

```diff
@@ -0,0 +1,74 @@
+from typing import Optional
+
+import tensorrt as trt
+from torch.fx.node import Target
+from torch_tensorrt.dynamo._SourceIR import SourceIR
+from torch_tensorrt.dynamo.conversion import impl
+from torch_tensorrt.dynamo.conversion._ConversionContext import ConversionContext
+from torch_tensorrt.dynamo.conversion.converter_utils import (
+    cast_trt_tensor,
+    flatten_dims,
+    get_axes_for_reduce_op,
+    get_positive_dim,
+)
+from torch_tensorrt.fx.converters.converter_utils import set_layer_name
+from torch_tensorrt.fx.types import TRTTensor
+
+
+def argmax(
+    ctx: ConversionContext,
+    target: Target,
+    source_ir: Optional[SourceIR],
+    name: str,
+    input: TRTTensor,
+    dim: Optional[int],
+    keep_dim: bool = False,
+) -> TRTTensor:
+    if input.dtype == trt.int32:
+        input = cast_trt_tensor(ctx, input, trt.float32, name, source_ir)
```

Comment:
It looks like missing `target`. The interface is:
```
def cast_trt_tensor(
    ctx: ConversionContext,
    input_val: TRTTensor,
    dtype: TRTDataType,
    name: str,
    target: Target = "",
    source_ir: Optional[SourceIR] = None,
) -> TRTTensor:
```

Reply:
Great catch - thanks!